lua/plenary/curl.lua

```diff
@@ -179,6 +180,15 @@ parse.accept_header = function(s)
   return { "-H", "Accept: " .. s }
 end
 
+parse.http_version = function(s)
+  if not s then
+    return
+  end
```

Comment:
do we want some kind of validation? that users can actually only pass in `HTTP/0.9`...

because right now we dont have that and unconditionally do the transformation